src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReportTest.java

```diff
@@ -1,7 +1,9 @@
 package uk.gov.hmcts.ethos.replacement.docmosis.reports.hearingsbyhearingtype;
 
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```

Comment:
Have you checked SonarLint?

Reply:
yeah fixed sonarLint issues, it is using now JUnit5 only